lxd/storage/backend_lxd_patches.go

```diff
@@ -190,6 +190,16 @@ func patchBucketNames(b *lxdBackend) error {
 		return err
 	}
 
+	if len(buckets) == 0 {
+		return nil
+	}
+
+	// ListVolumes assumes that the pool is mounted for dir & btrfs drivers
+	_, err = b.Mount()
```

Comment:
Should we unmount the pool afterwards (to ensure clean exit in case of an error)?

Also, this will ensure the directories are created for dir and btrfs, but have you maybe tested with zfs and lvm? Just to ensure there is no side effects?

Reply:
:+1: for unmount.